pkg/providers/docker/reconciler/reconciler.go

```diff
@@ -4,14 +4,27 @@ import (
 	"context"
 
 	"github.com/go-logr/logr"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 
 	anywherev1 "github.com/aws/eks-anywhere/pkg/api/v1alpha1"
+	c "github.com/aws/eks-anywhere/pkg/cluster"
```

Comment:
nit: can we keep this as `cluster`?
I find more readable to name the actual cluster object `c` in the method arguments than naming the package `c`.